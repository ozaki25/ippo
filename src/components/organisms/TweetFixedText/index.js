import React from 'react';
import { Typography } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';
import styled from 'styled-components';
import PinIcon from 'src/components/atoms/PinIcon';

const styles = theme => ({
  icon: {
    fontSize: '15px',
    margin: '3px',
  },
  text: {
    color: theme.palette.text.secondary,
  },
});

const Container = styled.div`
  display: flex;
  font-size: 15px;
  flex-direction: row;
  width: 100%;
`;

const IconContainer = styled.div`
  flex-grow: 1;
  max-width: 48px;
  margin: 0 5px;
  text-align: right;
`;

const TweetFixedText = ({ classes }) => (
  <Container>
    <IconContainer>
      <PinIcon color="primary" fontSize="small" className={classes.icon} />
    </IconContainer>
    <Typography className={classes.text}>固定されたツイート</Typography>
  </Container>
);

TweetFixedText.displayName = 'TweetFixedText';

TweetFixedText.propTypes = {};

TweetFixedText.defaultProps = {};

export default withStyles(styles)(TweetFixedText);
