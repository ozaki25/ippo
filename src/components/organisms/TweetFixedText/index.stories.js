import React from 'react';
import TweetFixedText from '.';

export default {
  title: 'organisms/TweetFixedText',
};

const props = {};

export const 通常パターン = () => <TweetFixedText {...props} />;
