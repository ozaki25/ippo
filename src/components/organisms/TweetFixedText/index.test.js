import React from 'react';
import { snapshot } from 'src/test/helpers';
import TweetFixedText from '.';

const props = {};

snapshot('TweetFixedText/nomal', <TweetFixedText {...props} />);
