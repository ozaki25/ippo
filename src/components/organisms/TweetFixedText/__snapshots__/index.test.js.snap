// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: TweetFixedText/nomal 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 15px;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  max-width: 48px;
  margin: 0 5px;
  text-align: right;
}

<MockRouter
  createHref={[Function]}
  location=""
  params={Object {}}
  path=""
  push={[Function]}
  replace={[Function]}
  url=""
>
  <WithStyles(TweetFixedText)>
    <TweetFixedText
      classes={
        Object {
          "icon": "TweetFixedText-icon-1",
          "text": "TweetFixedText-text-2",
        }
      }
    >
      <styled.div>
        <StyledComponent
          forwardedComponent={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "sc-bdVaJa",
                "isStatic": true,
                "lastClassName": "c0",
                "rules": Array [
                  "
  display: flex;
  font-size: 15px;
  flex-direction: row;
",
                ],
              },
              "displayName": "styled.div",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "sc-bdVaJa",
              "target": "div",
              "toString": [Function],
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            }
          }
          forwardedRef={null}
        >
          <div
            className="c0"
          >
            <styled.div>
              <StyledComponent
                forwardedComponent={
                  Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "sc-bwzfXH",
                      "isStatic": true,
                      "lastClassName": "c1",
                      "rules": Array [
                        "
  flex-grow: 1;
  max-width: 48px;
  margin: 0 5px;
  text-align: right;
",
                      ],
                    },
                    "displayName": "styled.div",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "sc-bwzfXH",
                    "target": "div",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  }
                }
                forwardedRef={null}
              >
                <div
                  className="c1"
                >
                  <PinIcon
                    className="TweetFixedText-icon-1"
                    color="primary"
                    fontSize="small"
                  >
                    <WithStyles(SvgIcon)
                      className="TweetFixedText-icon-1"
                      color="primary"
                      fontSize="small"
                    >
                      <SvgIcon
                        className="TweetFixedText-icon-1"
                        classes={
                          Object {
                            "colorAction": "MuiSvgIcon-colorAction-6",
                            "colorDisabled": "MuiSvgIcon-colorDisabled-8",
                            "colorError": "MuiSvgIcon-colorError-7",
                            "colorPrimary": "MuiSvgIcon-colorPrimary-4",
                            "colorSecondary": "MuiSvgIcon-colorSecondary-5",
                            "fontSizeInherit": "MuiSvgIcon-fontSizeInherit-9",
                            "fontSizeLarge": "MuiSvgIcon-fontSizeLarge-11",
                            "fontSizeSmall": "MuiSvgIcon-fontSizeSmall-10",
                            "root": "MuiSvgIcon-root-3",
                          }
                        }
                        color="primary"
                        component="svg"
                        fontSize="small"
                        viewBox="0 0 24 24"
                      >
                        <svg
                          aria-hidden="true"
                          className="MuiSvgIcon-root-3 MuiSvgIcon-colorPrimary-4 MuiSvgIcon-fontSizeSmall-10 TweetFixedText-icon-1"
                          focusable="false"
                          role="presentation"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19.083 1c0 1.018-1.424 1.907-3.541 2.382V11c2.926.652 4.958 2.086 4.958 3.751h-7.792V23h-1.416v-8.25H3.5c0-1.665 2.032-3.1 4.958-3.751V3.382C6.341 2.907 4.917 2.018 4.917 1h14.166z"
                          />
                           
                        </svg>
                      </SvgIcon>
                    </WithStyles(SvgIcon)>
                  </PinIcon>
                </div>
              </StyledComponent>
            </styled.div>
            <WithStyles(Typography)
              className="TweetFixedText-text-2"
            >
              <Typography
                align="inherit"
                className="TweetFixedText-text-2"
                classes={
                  Object {
                    "alignCenter": "MuiTypography-alignCenter-35",
                    "alignJustify": "MuiTypography-alignJustify-37",
                    "alignLeft": "MuiTypography-alignLeft-34",
                    "alignRight": "MuiTypography-alignRight-36",
                    "body1": "MuiTypography-body1-21",
                    "body2": "MuiTypography-body2-20",
                    "button": "MuiTypography-button-23",
                    "caption": "MuiTypography-caption-22",
                    "colorError": "MuiTypography-colorError-46",
                    "colorInherit": "MuiTypography-colorInherit-41",
                    "colorPrimary": "MuiTypography-colorPrimary-42",
                    "colorSecondary": "MuiTypography-colorSecondary-43",
                    "colorTextPrimary": "MuiTypography-colorTextPrimary-44",
                    "colorTextSecondary": "MuiTypography-colorTextSecondary-45",
                    "display1": "MuiTypography-display1-16",
                    "display2": "MuiTypography-display2-15",
                    "display3": "MuiTypography-display3-14",
                    "display4": "MuiTypography-display4-13",
                    "gutterBottom": "MuiTypography-gutterBottom-39",
                    "h1": "MuiTypography-h1-24",
                    "h2": "MuiTypography-h2-25",
                    "h3": "MuiTypography-h3-26",
                    "h4": "MuiTypography-h4-27",
                    "h5": "MuiTypography-h5-28",
                    "h6": "MuiTypography-h6-29",
                    "headline": "MuiTypography-headline-17",
                    "noWrap": "MuiTypography-noWrap-38",
                    "overline": "MuiTypography-overline-32",
                    "paragraph": "MuiTypography-paragraph-40",
                    "root": "MuiTypography-root-12",
                    "srOnly": "MuiTypography-srOnly-33",
                    "subheading": "MuiTypography-subheading-19",
                    "subtitle1": "MuiTypography-subtitle1-30",
                    "subtitle2": "MuiTypography-subtitle2-31",
                    "title": "MuiTypography-title-18",
                  }
                }
                color="default"
                gutterBottom={false}
                headlineMapping={
                  Object {
                    "body1": "p",
                    "body2": "p",
                    "display1": "h1",
                    "display2": "h1",
                    "display3": "h1",
                    "display4": "h1",
                    "h1": "h1",
                    "h2": "h2",
                    "h3": "h3",
                    "h4": "h4",
                    "h5": "h5",
                    "h6": "h6",
                    "headline": "h1",
                    "subheading": "h3",
                    "subtitle1": "h6",
                    "subtitle2": "h6",
                    "title": "h2",
                  }
                }
                noWrap={false}
                paragraph={false}
                theme={
                  Object {
                    "breakpoints": Object {
                      "between": [Function],
                      "down": [Function],
                      "keys": Array [
                        "xs",
                        "sm",
                        "md",
                        "lg",
                        "xl",
                      ],
                      "only": [Function],
                      "up": [Function],
                      "values": Object {
                        "lg": 1280,
                        "md": 960,
                        "sm": 600,
                        "xl": 1920,
                        "xs": 0,
                      },
                      "width": [Function],
                    },
                    "direction": "ltr",
                    "mixins": Object {
                      "gutters": [Function],
                      "toolbar": Object {
                        "@media (min-width:0px) and (orientation: landscape)": Object {
                          "minHeight": 48,
                        },
                        "@media (min-width:600px)": Object {
                          "minHeight": 64,
                        },
                        "minHeight": 56,
                      },
                    },
                    "overrides": Object {},
                    "palette": Object {
                      "action": Object {
                        "active": "rgba(0, 0, 0, 0.54)",
                        "disabled": "rgba(0, 0, 0, 0.26)",
                        "disabledBackground": "rgba(0, 0, 0, 0.12)",
                        "hover": "rgba(0, 0, 0, 0.08)",
                        "hoverOpacity": 0.08,
                        "selected": "rgba(0, 0, 0, 0.14)",
                      },
                      "augmentColor": [Function],
                      "background": Object {
                        "default": "#fafafa",
                        "paper": "#fff",
                      },
                      "common": Object {
                        "black": "#000",
                        "white": "#fff",
                      },
                      "contrastThreshold": 3,
                      "divider": "rgba(0, 0, 0, 0.12)",
                      "error": Object {
                        "contrastText": "#fff",
                        "dark": "#d32f2f",
                        "light": "#e57373",
                        "main": "#f44336",
                      },
                      "getContrastText": [Function],
                      "grey": Object {
                        "100": "#f5f5f5",
                        "200": "#eeeeee",
                        "300": "#e0e0e0",
                        "400": "#bdbdbd",
                        "50": "#fafafa",
                        "500": "#9e9e9e",
                        "600": "#757575",
                        "700": "#616161",
                        "800": "#424242",
                        "900": "#212121",
                        "A100": "#d5d5d5",
                        "A200": "#aaaaaa",
                        "A400": "#303030",
                        "A700": "#616161",
                      },
                      "primary": Object {
                        "contrastText": "#fff",
                        "dark": "#303f9f",
                        "light": "#7986cb",
                        "main": "#3f51b5",
                      },
                      "secondary": Object {
                        "contrastText": "#fff",
                        "dark": "#c51162",
                        "light": "#ff4081",
                        "main": "#f50057",
                      },
                      "text": Object {
                        "disabled": "rgba(0, 0, 0, 0.38)",
                        "hint": "rgba(0, 0, 0, 0.38)",
                        "primary": "rgba(0, 0, 0, 0.87)",
                        "secondary": "rgba(0, 0, 0, 0.54)",
                      },
                      "tonalOffset": 0.2,
                      "type": "light",
                    },
                    "props": Object {},
                    "shadows": Array [
                      "none",
                      "0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12)",
                      "0px 1px 5px 0px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 3px 1px -2px rgba(0,0,0,0.12)",
                      "0px 1px 8px 0px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 3px 3px -2px rgba(0,0,0,0.12)",
                      "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
                      "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
                      "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
                      "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
                      "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
                      "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
                      "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
                      "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
                      "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
                      "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
                      "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
                      "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
                      "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
                      "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
                      "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
                      "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
                      "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
                      "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
                      "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
                      "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
                      "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
                    ],
                    "shape": Object {
                      "borderRadius": 4,
                    },
                    "spacing": Object {
                      "unit": 8,
                    },
                    "transitions": Object {
                      "create": [Function],
                      "duration": Object {
                        "complex": 375,
                        "enteringScreen": 225,
                        "leavingScreen": 195,
                        "short": 250,
                        "shorter": 200,
                        "shortest": 150,
                        "standard": 300,
                      },
                      "easing": Object {
                        "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
                        "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
                        "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
                        "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
                      },
                      "getAutoHeightDuration": [Function],
                    },
                    "typography": Object {
                      "body1": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 400,
                        "lineHeight": "1.46429em",
                      },
                      "body1Next": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.00938em",
                        "lineHeight": 1.5,
                      },
                      "body2": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 500,
                        "lineHeight": "1.71429em",
                      },
                      "body2Next": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.01071em",
                        "lineHeight": 1.5,
                      },
                      "button": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 500,
                        "textTransform": "uppercase",
                      },
                      "buttonNext": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 500,
                        "letterSpacing": "0.02857em",
                        "lineHeight": 1.3125,
                        "textTransform": "uppercase",
                      },
                      "caption": Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.75rem",
                        "fontWeight": 400,
                        "lineHeight": "1.375em",
                      },
                      "captionNext": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.75rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.03333em",
                        "lineHeight": 1.66,
                      },
                      "display1": Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "2.125rem",
                        "fontWeight": 400,
                        "lineHeight": "1.20588em",
                      },
                      "display2": Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "2.8125rem",
                        "fontWeight": 400,
                        "lineHeight": "1.13333em",
                        "marginLeft": "-.02em",
                      },
                      "display3": Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "3.5rem",
                        "fontWeight": 400,
                        "letterSpacing": "-.02em",
                        "lineHeight": "1.30357em",
                        "marginLeft": "-.02em",
                      },
                      "display4": Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "7rem",
                        "fontWeight": 300,
                        "letterSpacing": "-.04em",
                        "lineHeight": "1.14286em",
                        "marginLeft": "-.04em",
                      },
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": 14,
                      "fontWeightLight": 300,
                      "fontWeightMedium": 500,
                      "fontWeightRegular": 400,
                      "h1": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "6rem",
                        "fontWeight": 300,
                        "letterSpacing": "-0.01562em",
                        "lineHeight": 1,
                      },
                      "h2": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "3.75rem",
                        "fontWeight": 300,
                        "letterSpacing": "-0.00833em",
                        "lineHeight": 1,
                      },
                      "h3": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "3rem",
                        "fontWeight": 400,
                        "letterSpacing": "0em",
                        "lineHeight": 1.04,
                      },
                      "h4": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "2.125rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.00735em",
                        "lineHeight": 1.17,
                      },
                      "h5": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1.5rem",
                        "fontWeight": 400,
                        "letterSpacing": "0em",
                        "lineHeight": 1.33,
                      },
                      "h6": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1.25rem",
                        "fontWeight": 500,
                        "letterSpacing": "0.0075em",
                        "lineHeight": 1.6,
                      },
                      "headline": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1.5rem",
                        "fontWeight": 400,
                        "lineHeight": "1.35417em",
                      },
                      "overline": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.75rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.08333em",
                        "lineHeight": 2.66,
                        "textTransform": "uppercase",
                      },
                      "pxToRem": [Function],
                      "round": [Function],
                      "subheading": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1rem",
                        "fontWeight": 400,
                        "lineHeight": "1.5em",
                      },
                      "subtitle1": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1rem",
                        "fontWeight": 400,
                        "letterSpacing": "0.00938em",
                        "lineHeight": 1.75,
                      },
                      "subtitle2": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "0.875rem",
                        "fontWeight": 500,
                        "letterSpacing": "0.00714em",
                        "lineHeight": 1.57,
                      },
                      "title": Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                        "fontSize": "1.3125rem",
                        "fontWeight": 500,
                        "lineHeight": "1.16667em",
                      },
                      "useNextVariants": false,
                    },
                    "zIndex": Object {
                      "appBar": 1100,
                      "drawer": 1200,
                      "mobileStepper": 1000,
                      "modal": 1300,
                      "snackbar": 1400,
                      "tooltip": 1500,
                    },
                  }
                }
              >
                <p
                  className="MuiTypography-root-12 MuiTypography-body1-21 TweetFixedText-text-2"
                >
                  固定されたツイート
                </p>
              </Typography>
            </WithStyles(Typography)>
          </div>
        </StyledComponent>
      </styled.div>
    </TweetFixedText>
  </WithStyles(TweetFixedText)>
</MockRouter>
`;
